Update guidance for personal statements (#4198)
DFE-Digital/get-into-teaching-app: app/views/content/how-to-apply-for-teacher-training/teacher-training-personal-statement.md

Your personal statement can be up to 1000 words. 90% of successful candidates write 500 words or more.

## What should I write about in my personal statement?

Your personal statement should include:

> * why you want to be a teacher

Author: I can't see one?

> * why you want to teach the subject or specialism you're applying - for primary, describe why you want to teach this age group  

Author: ```suggestion
* why you want to teach the subject or specialism you're applying for - if you're applying for primary, describe why you want to teach this age group
```

* the experience and personal characteristics you have, and why they would make you a great teacher 

> You can include any information you think is relevant to demonstrate your knowledge, qualifications, and experience. Teacher training providers want to see your passion and that you understand the bigger picture of teaching.

Author: ```suggestion
You can include any information you think is relevant to demonstrate your knowledge, qualifications, and experience. Teacher training providers want to see your passion and that you understand what a career in teaching involves.
```
